dbbackup/tests/settings.py

```diff
@@ -3,6 +3,12 @@
 """
 import os
 import tempfile
+import sys
+from dotenv import load_dotenv
```

Comment:
Maybe we should use django-database-url ?

Reply:
Using all the separate components actually works pretty well for testing in this case, as you can test any possible edge cases.